test/vanilla/ignite-generate.test.ts

```diff
@@ -91,15 +91,15 @@ test(\\"can be created\\", () => {
 "
 `)
       expect(read(`${TEMP_DIR}/app/models/index.ts`)).toMatchInlineSnapshot(`
-        "export * from \\"./RootStore\\"
-        export * from \\"./helpers/getRootStore\\"
-        export * from \\"./helpers/useStores\\"
-        export * from \\"./helpers/setupRootStore\\"
+"export * from "./RootStore"
+export * from "./helpers/getRootStore"
+export * from "./helpers/useStores"
+export * from "./helpers/setupRootStore"
 
-        // @mst remove-file
-        export * from \\"./Pizza\\"
-        "
-      `)
+// @mst remove-file
+export * from "./Pizza"
+"
+`)
       expect(read(`${TEMP_DIR}/app/models/RootStore.ts`)).toEqual(
```

Comment:
I'm also wondering if the eslint config change will have impact on these tests - but CI is green so seems fine